src/Connection.cpp

```diff
@@ -21,8 +21,9 @@ Connection::Connection(const std::string &uri)
   // "radio://0/80/2M/E7E7E7E7E7" -> connect over radio
   // "radio://*/80/2M/E7E7E7E7E7" -> auto-pick radio
   // "radio://*/80/2M/*" -> broadcast/P2P sniffing on channel 80
```

Comment:
I don't think using this /broadcast URI is compatible with the overall URI design. The star as suggested here might be another option, apart from what's discussed in issue #8.